packages/firestore/src/platform/platform.ts

```diff
@@ -43,6 +43,12 @@ export interface Platform {
   /** Converts a binary string to a Base64 encoded string. */
   btoa(raw: string): string;
 
+  /**
+   * Generates a random byte `b` with `0 <= b <= max`. If `max` is negative,
+   * it is the same as setting it to byte max (255).
+   */
+  randomByte(max: number): number;
```

Comment:
The platforms both provide ways to get an array of random bytes. It's better to keep this interface structured in terms of exposing common platform primitives and than we can implement any derived concerns (like limiting to a max value just once).

Concretely, I think this should be `randomBytes` that takes a number of bytes to generate and returns a `Uint8Array` of those contents. (Note that Node's `crypto.randomBytes` returns a `Buffer` but that's a subclass of `Uint8Array` so it's a suitable return value).

Reply:
Done.